Omargh/Tasks.py

```diff
@@ -0,0 +1,26 @@
+from typing import Dict, Optional
+from modules import TaskCreate, Task, UrgentTask
+from uuid import UUID, uuid4
+class Task:
+    task_id=0
+    def __init__(self,title,description,due_date,status="InProgress"):
+        Task.task_id+=1
```

Comment:
please use the uuid instead 

look here: https://docs.python.org/3/library/uuid.html
use uuid4

Reply:
Done